test/development/app-dir/prerender-indicator/app/force-dynamic/page.tsx

```diff
@@ -0,0 +1,5 @@
+import { WaitForHydration } from './wait-for-hydration'
+
```

Comment:
Shouldn't we have force-dynamic here?

Reply:
Woops, yes -- accidentally removed when refactoring 🙈